doc/source/reference/c-api/coremath.rst

```diff
@@ -249,6 +249,11 @@ Linking against the core math library in an extension
 To use the core math library in your own extension, you need to add the npymath
 compile and link options to your extension in your setup.py:
 
+	.. hidden in a comment so as to be included in refguide but not rendered documentation
+	  >>> import numpy.distutils.misc_util
+	  >>> config = np.distutils.misc_util.Configuration(None, '', '.')
+	  >>> with open('foo.c', 'w') as f: pass
+
         >>> from numpy.distutils.misc_util import get_info
```

Comment:
Indents look odd here - you have:
```
.. the comment
  >>> the hidden code
 >>> the real code
```

Reply:
Mixed tabs and spaces

Reply:
This looks a bit less surprising and seems to work. Blank line is important.
```suggestion
.. hidden in a comment so as to be included in refguide but not rendered documentation
        >>> import numpy.distutils.misc_util
        >>> config = np.distutils.misc_util.Configuration(None, '', '.')
        >>> with open('foo.c', 'w') as f: pass
..

        >>> from numpy.distutils.misc_util import get_info
```

Reply:
This does not pass refguide_check

Reply:
Hmm, do you know why? Is it finding the `..` and treating it as output? Would an extra blank line solve it?

Reply:
"inconsistent leading whitespace", doctest does not accept the indent.

Extra blank lines did not help, I tried that.

```pytb
Checking files in doc/source/reference/c-api/coremath.rst:
doc/source/reference/c-api/coremath.rst Traceback (most recent call last):
  File "tools/refguide_check.py", line 1232, in <module>
    main(argv=sys.argv[1:])
  File "tools/refguide_check.py", line 1188, in main
    check_documentation(rst_path, results, args, dots)
  File "tools/refguide_check.py", line 1074, in check_documentation
    doctest_warnings=args.doctest_warnings)
  File "tools/refguide_check.py", line 983, in check_doctests_testfile
    tests = parser.get_doctest(part, ns, fname, fname, base_line_no)
  File "/usr/lib/python3.7/doctest.py", line 668, in get_doctest
    return DocTest(self.get_examples(string, name), globs,
  File "/usr/lib/python3.7/doctest.py", line 682, in get_examples
    return [x for x in self.parse(string, name)
  File "/usr/lib/python3.7/doctest.py", line 644, in parse
    self._parse_example(m, name, lineno)
  File "/usr/lib/python3.7/doctest.py", line 714, in _parse_example
    lineno + len(source_lines))
  File "/usr/lib/python3.7/doctest.py", line 800, in _check_prefix
    (lineno+i+1, name, line))
ValueError: line 256 of the docstring for doc/source/reference/c-api/coremath.rst has inconsistent leading whitespace: '..'
```